parse.y

```diff
@@ -565,13 +579,15 @@ yylex(void)
 		{ "created",		CREATED },
 		{ "date",		DATE },
 		{ "discard",		DISCARD },
+		{ "draft",		DRAFT },
 		{ "exec",		EXEC },
 		{ "flag",		FLAG },
 		{ "header",		HEADER },
 		{ "isdirectory",	ISDIRECTORY },
 		{ "label",		LABEL },
 		{ "maildir",		MAILDIR },
 		{ "match",		MATCH },
+		{ "mdflags",		MDFLAGS },
```

Comment:
Please call the new action `flags` and make sure to update the expr_type and eval routine accordingly.

Reply:
"flags" sounds more natural than "mdflags" (4 consonants!)